src/data-catalog/DataBrowserDetails.ts

```diff
@@ -119,14 +148,14 @@ const MainContent = ({ dataObj }) => {
 export const SidebarComponent = ({ dataObj, id }) => {
   const [feedbackShowing, setFeedbackShowing] = useState(false);
   const [datasetNotSupportedForExport, setDatasetNotSupportedForExport] = useState(false);
-  const [snapshotExportJobId, setSnapshotExportJobId] = useState();
+  const [snapshotExportJobId, setSnapshotExportJobId] = useState<string>();
   const [tdrSnapshotPreparePolling, setTdrSnapshotPreparePolling] = useState(false);
   const sidebarButtonWidth = 230;
   const access = getDatasetAccessType(dataObj);
   const actionTooltip = access === datasetAccessTypes.Granted ? '' : uiMessaging.controlledFeatureTooltip;
 
   const importDataToWorkspace = (dataset) => {
-    Ajax().Metrics.captureEvent(`${Events.catalogWorkspaceLink}:detailsView`, {
+    Ajax().Metrics.captureEvent(Events.catalogWorkspaceLink, {
       id,
       title: dataObj['dct:title'],
       source: dataset['dcat:accessURL'],
```

Comment:
what are dct and dcat?

Reply:
They're properties from the Terra Interoperability Model (https://datamodel.terra.bio/TerraCoreDataModel.html) - we've kinda abandoned it but it was used for a lot of catalog datasets properties.

Reply:
These are in TIM but they're inherited from w3c DCAT ontology https://www.w3.org/TR/vocab-dcat-3/